Documentation/gettingstarted/istio.rst

```diff
@@ -99,6 +97,18 @@ Deploy Istio onto Kubernetes:
 ::
 
     $ kubectl create namespace istio-system
+    $ helm template ${ISTIO_HOME}/install/kubernetes/helm/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
+
+Verify that 53 Istio CRDs have been created:
+
+::
+
+    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
```

Comment:
Would be nice to have something like `watch -n 1 "kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l"` instead so users don't have to keep entering the same command again and again.

Reply:
I'll try that and update accordingly!